packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -258,11 +258,7 @@ class PerformanceMonitoring {
       tr.resetSpans()
     }
 
-    const checkBrowserResponsiveness = this._configService.get(
-      'checkBrowserResponsiveness'
-    )
-
-    if (checkBrowserResponsiveness && tr.options.checkBrowserResponsiveness) {
+    if (tr.options.checkBrowserResponsiveness) {
```

Comment:
Instead of this, we can use the counter itself to check whether the transaction has passed an interval. 

`if (tr. browserResponsivenessCounter > 0)`

Reply:
We can't rely on `browserResponsivenessCounter ` since it's only updated in 500ms intervals. see my main comment